crates/bitwarden-exporters/src/export.rs

```diff
@@ -41,3 +48,70 @@ pub(crate) fn export_organization_vault(
 ) -> Result<String, ExportError> {
     todo!();
 }
+
+/// See [crate::ClientExporters::export_cxf] for more documentation.
+pub(crate) fn export_cxf(
+    client: &Client,
+    account: Account,
+    ciphers: Vec<Cipher>,
+) -> Result<String, ExportError> {
+    let enc = client.internal.get_encryption_settings()?;
+
+    let ciphers: Vec<crate::Cipher> = ciphers
+        .into_iter()
+        .flat_map(|c| crate::Cipher::from_cipher(&enc, c))
+        .collect();
+
+    Ok(build_cxf(account, ciphers)?)
+}
+
+fn encrypt_import(enc: &dyn KeyContainer, cipher: ImportingCipher) -> Result<Cipher, ExportError> {
+    let mut view: CipherView = cipher.clone().into();
+
+    // Get passkey from cipher if cipher is type login
+    let passkey = match cipher.r#type {
+        crate::CipherType::Login(login) => login.fido2_credentials,
+        _ => None,
+    };
+
+    if let Some(passkey) = passkey {
+        let passkeys: Vec<Fido2CredentialFullView> = passkey
+            .into_iter()
+            .map(|p| Fido2CredentialFullView {
+                credential_id: p.credential_id,
+                key_type: p.key_type,
+                key_algorithm: p.key_algorithm,
+                key_curve: p.key_curve,
+                key_value: p.key_value,
+                rp_id: p.rp_id,
+                user_handle: p.user_handle,
+                user_name: p.user_name,
+                counter: p.counter.to_string(),
+                rp_name: p.rp_name,
+                user_display_name: p.user_display_name,
+                discoverable: p.discoverable,
+                creation_date: p.creation_date,
+            })
```

Comment:
Would this be better off in it's own separate `From<Fido2Credential>` implementation?